### ISSUE TITLE
Allow multiline text entry for changes and eliminate problem with repeated output to terminal

### ISSUE THREAD

**Issue by sheldonhull** [miniscruff/changie#385]
**Is your feature request related to a problem? Please describe.**

The input for a change might contain more complex changes.
I'd like to write multiple lines.

**Describe the solution you'd like**

The body should be a multi-line input and this should be saved via 

```yaml
---
body: |
  my content
  
  - Item 1
  - item 2
```

**Describe alternatives you've considered**

I use [gum](https://github.com/charmbracelet/gum#write) for this. 

![2022-10-21-14 44 39-Code- changie yaml — magetools](https://user-images.githubusercontent.com/3526320/197277087-39553112-67fc-4d29-a78e-2b38ddd3dfad.gif)

**Additional context**

Currently the input once exceeding the width of the terminal incorrectly reflects repeated lines. See attached gif. 

![2022-10-21-14 37 19-Code- changie yaml — magetools](https://user-images.githubusercontent.com/3526320/197276853-759239ff-141a-4f2e-8c79-6d1b369d0903.gif)

**Comment by miniscruff** [comment]
Multi-line input is going to be an option with survey once this PR is merged and I make a new custom type: https://github.com/miniscruff/changie/pull/374
I will also probably add some body config to switch from a normal input to multi-line.

Currently the only way you can do it is by adding custom choices for what each "line" would be and combine it together. You can specify it per kind as well if you want it only for breaking changes or whatever.

tl;dr this is more a limitation of promptui at this point.

**Comment by miniscruff** [comment]
@sheldonhull Just want to give an update on this, it looks like a survey maintainer has taken a look at the windows issue and has a draft PR open. Once that is merged I will swap from promptui to survey and add a multiline option soon after that.

**Comment by sheldonhull** [comment]
FYI the bubbletea based gum cli worked well in powershell and windows with the multi-line edit. Survey is great as well, just wanted to mention that if you mix and match I've found bubbletea + another library this last week that are designed for robust text input and offer multi-line. If you end up not getting progress on the survey side I'd be glad to provide a few examples with the other resources I've come across. 

Definitely appreciating changie! 

Off topic question: Have you considered a command like `changie batch patch` or such that would allow bumping just to next calculated version without needing to specify the patch version? Can open a new discussion on that later if you think it's interesting.

**Comment by miniscruff** [comment]
Work will begin soon using a new tool related to this issue: https://github.com/miniscruff/changie/issues/457

**Comment by miniscruff** [comment]
Support was added to v1.12.0, let me know if any bugs appear
